web-ts/src/testing/mockUseLoadFromEndpoint.tsx

```diff
@@ -0,0 +1,29 @@
+import { useLoadFromEndpoint } from 'api-client'
```

Comment:
I think this makes more sense in the api-client as `useLoadFromEndpoint` is also imported from there.